src/services/contracts/safeContracts.ts

```diff
@@ -170,3 +172,16 @@ export const getFallbackHandlerContractInstance = (chainId: string): Compatibili
 
   return new Contract(contractAddress, new Interface(fallbackHandlerDeployment.abi)) as Compatibility_fallback_handler
 }
+
+// Sign messages deployment
+// TODO: Should this be implemented in Core SDK?
+export const getSignMessageLibDeploymentContractInstance = (chainId: string): Sign_message_lib => {
+  const signMessageLibDeployment = getSignMessageLibDeployment({ network: chainId }) || getSignMessageLibDeployment()
```

Comment:
Not directly related to the PR, but I wonder why we need this fallback logic. Do we not trust that `({ network: chainId })` filter works? cc @katspaugh @iamacook

Reply:
I agree, but also at the same time, it looks like removing it won't break anything